docs/UserGuide.md

```diff
@@ -42,17 +42,18 @@ FriendFolio is a **desktop CLI (Command Line Interface)-optimized app** with int
     1. Dashboard (Coming Soon)
     2. Finding Availabilities (Coming Soon)
     3. Commands
-        1. [`Help` Command](#viewing-help-help)
-        2. [`Add` Command](#adding-a-person-add)
-        3. [`List` Command](#listing-all-persons-list)
-        4. [`Edit` Command](#editing-a-person-edit)
-        5. [`Find` Command](#locating-persons-by-name-find)
-        6. [`Delete` Command](#deleting-a-person-delete)
-        7. [`Split` Command](#splitting-an-amount-owed-split)
-        8. [`Sort` Command](#sorting-contacts-sort)
-        9. [`Pay` Command](#generating-payment-qr-code--pay)
-        10. [`Clear` Command](#clearing-all-entries-clear)
-        11. [`Exit` Command](#exiting-the-program-exit)
+        1. [`Help` Command](#viewing-help--help)
+        2. [`Add` Command](#adding-a-person--add)
+        3. [`List` Command](#listing-all-persons--list)
+        4. [`Edit` Command](#editing-a-person--edit)
+        5. [`Find` Command](#locating-persons-by-name--find)
```

Comment:
We don't have find anymore because we combined with filter, can help remove?

Reply:
removed!